mobile/lib/main.dart

```diff
@@ -170,6 +180,7 @@ class MainApp extends StatelessWidget {
               ],
             ),
           ),
+          bottomNavigationBar: const RaBottomNavigationBar(currentIndex: 0),
```

Comment:
if currentIndex is implemented with useState, parameter here is unnecessary
```suggestion
          bottomNavigationBar: const RaBottomNavigationBar(),
```